### ISSUE TITLE
Unseen message with listMailbox && rename mailbox ?

### ISSUE THREAD

**Issue by hoangthienclub** [emailjs/emailjs-imap-client#145]
Do you response unseen message with folder mailbox and support api rename mailbox

**Comment by nifgraup** [comment]
@hoangthienclub I don't understand the question, could you rephrase it?

**Comment by hoangthienclub** [comment]
Support total unseen message API get mailbox. Sorry cause i'm bad english

**Comment by nifgraup** [comment]
Sounds like you want to count messages without \Seen flags in a mailbox? Try https://github.com/emailjs/emailjs-imap-client#searching and check the length of the resulting array.
